airbyte-integrations/connectors/source-postgres/src/main/java/io/airbyte/integrations/source/postgres/ctid/PostgresCtidHandler.java

```diff
@@ -165,21 +167,20 @@ private AutoCloseableIterator<AirbyteMessage> augmentWithLogs(final AutoCloseabl
   }
 
   private AutoCloseableIterator<AirbyteMessage> augmentWithState(final AutoCloseableIterator<AirbyteMessageWithCtid> recordIterator,
-                                                                 final AirbyteStreamNameNamespacePair pair) {
+                                                                 final ConfiguredAirbyteStream airbyteStream) {
 
-    final CtidStatus currentCtidStatus = ctidStateManager.getCtidStatus(pair);
-    final JsonNode incrementalState =
-        (currentCtidStatus == null || currentCtidStatus.getIncrementalState() == null) ? streamStateForIncrementalRunSupplier.apply(pair)
-            : currentCtidStatus.getIncrementalState();
     final Duration syncCheckpointDuration =
         config.get(SYNC_CHECKPOINT_DURATION_PROPERTY) != null ? Duration.ofSeconds(config.get(SYNC_CHECKPOINT_DURATION_PROPERTY).asLong())
-            : CtidStateIterator.SYNC_CHECKPOINT_DURATION;
+            : DebeziumIteratorConstants.SYNC_CHECKPOINT_DURATION;
```

Comment:
nit : perhaps we should create a new InitialLoadConstants.SYNC_CHECKPOINT_DURATIONS to have the ability to differentiate between different checkpointing constants?

Reply:
Is there a difference for doing initial load vs regular incremental load? Feels like we should do that only if we found that would be required.

Reply:
No difference right now. I think we don't need to create additional ones. But maybe a todo or something to rename the file in `DebeziumIteratorConstants` to just `IteratorConstants`

Reply:
added a to do. Will move after I checked in other iterators.